### DISCUSSION TITLE
Can we remove the `tabs` permission?

### DISCUSSION THREAD

**Discussion by gera2ld** [violentmonkey/violentmonkey#1818] [General]
Original message: https://discord.com/channels/995346102003965952/995346102461157378/1104049389250551899

It is described as "Read your browsing history" which looks scary.

**Comment by tophf** [comment]
We need it to handle the built-in pages in the browser like the newtab (start page) when we open the dashboard/editor. 

The phrasing of this permission warning in Chrome is misleading and actually wrong because we can't access the history in its conventional sense (the past history) via this permission, we can only observe navigation but it's also possible without this permission by any extension with broad host permissions such as ours.

Firefox doesn't display this warning arguably because they understand the above.
I think the concerned users should urge Chrome developers to fix the incorrect phrasing.
